OctoPrint/OctoPrint
# Fix - File upload error messages broken due to missing attribute in StorageError

#### What does this PR do and why is it necessary?

This PR re-adds the `message` attribute to the `StorageError` Class.

The attribute was deleted in #4625, but was still referenced in the file upload error handling, which generated another error.

References to StorageError.message:
- [files.py#L1293](https://github.com/OctoPrint/OctoPrint/blob/0addd657648bcfa65d553ba08800a26ff6072a79/src/octoprint/server/api/files.py#L1293) `f"{error.message} {error.code}", exc_info=error.cause`
- [files.py#L1314](https://github.com/OctoPrint/OctoPrint/blob/0addd657648bcfa65d553ba08800a26ff6072a79/src/octoprint/server/api/files.py#L1314) `abort(500, description=error.message)`


#### How was it tested? How can it be tested by the reviewer?

- Try to upload a file with a invalid file type.
- Observe terminal output:
  ```
  File "C:\Users\Kestin\Projects\OctoPrint\src\octoprint\filemanager\storage.py", line 871, in add_file
  raise StorageError(
    octoprint.filemanager.storage.StorageError: config.json is an unrecognized file type

  During handling of the above exception, another exception occurred:

  Traceback (most recent call last):

  ...

  File "C:\Users\Kestin\Projects\OctoPrint\src\octoprint\server\api\files.py", line 1293, in _abortWithException
  f"{error.message}: {error.code}", exc_info=error.cause
      ^^^^^^^^^^^^^
  AttributeError: 'StorageError' object has no attribute 'message'
  ```

#### What are the relevant tickets if any?

None that I know of.
